torchvision/prototype/transforms/functional/_type_conversion.py

```diff
@@ -34,13 +34,4 @@ def to_image_tensor(image: Union[torch.Tensor, PIL.Image.Image, np.ndarray], cop
     return _F.pil_to_tensor(image)
 
 
-def to_image_pil(
-    image: Union[torch.Tensor, PIL.Image.Image, np.ndarray], mode: Optional[str] = None
-) -> PIL.Image.Image:
-    if isinstance(image, PIL.Image.Image):
-        if mode != image.mode:
-            return image.convert(mode)
-        else:
-            return image
-
-    return _F.to_pil_image(image, mode=mode)
+to_image_pil = _F.to_pil_image
```

Comment:
Here is we removed the option to pass PIL Image and do a no-op. In above method `to_image_tensor` it is possible to pass torch tensor and do a no-op.

Reply:
Shall I remove also from `to_image_tensor()` the following?
```
    if isinstance(image, torch.Tensor):
        if copy:
            return image.clone()
        else:
            return image
```

In that case I would also need to lose the `copy`.

Reply:
I've updated this on the latest commit.